tests/fsharp/Compiler/Utilities.fs

```diff
@@ -0,0 +1,142 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+module FSharp.Compiler.UnitTests.Utilities
+
+open System
+open System.IO
+open System.Collections.Immutable
+open Microsoft.CodeAnalysis
+open Microsoft.CodeAnalysis.CSharp
+
+// This file mimics how Roslyn handles their compilation references for compilation testing
+
+[<RequireQualifiedAccess>]
+type TargetFramework =
+    | NetStandard20
+    | NetCoreApp30
```

Comment:
We might want add NetStandard21 as well.

Reply:
Not yet. The test resources from Roslyn to make this work does not include NetStandard21 yet.

Reply:
Ahhh .. that makes sense